docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+| `/status`               | `husky_msgs/HuskyStatus` | Displays system status information                                                                                                 |
+| `/estop`                | `std_msgs/Bool`          | Displays the estop status                                                                                                          |
+| `/odometry/filtered`    | `nav_msgs/Odometry`      | The odometry estimate of the robot from \lstinline!/ekf_localization                                                               |
+
+##### Motion Topics    {#motion-topics}
+
+| Motion Topics                       | `twist_mux` Priority | Description                                                        |
+| :---------------------------------- | :------------------- | :----------------------------------------------------------------- |
+| `husky_velocity_controller/cmd_vel` | -                    | Receives motion commands from `twist_mux` based off their priority |
+| `joy_teleop/cmd_vel`                | 10                   | Joystick teleop input                                              |
+| `twist_marker_server/cmd_vel`       | 8                    | Interactive marker teleop input                                    |
+| `move_base/cmd_vel`                 | 2                    | Autonomous movement input, for the Husky navigation packages       |
+| `cmd_vel`                           | 1                    | Miscellaneous external input                                       |
+
+---
+
+## Husky Software Setup   {#husky-software-setup}
+
+### Backing Up Robot Configuration
+
+Upgrading your Clearpath Husky to ROS Noetic from older ROS distributions is a straightforward process;
+however it's important to understand that each Husky is different, having undergone customization to your
+specifications.  For more complete upgrade instructions see
+[this guide](https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html).
+
+Please take the time to understand what these modifications are, and how to recreate them on your fresh install of
+Ubuntu Focal/ROS Noetic.
+
+#### Performing a Backup    {#performing-a-backup}
+
+1.  As a fail-safe, please make an image of your robot's hard drive. You should always be able to restore this image
+    if you need to revert back to your previous configuration.
+
+    - The easiest approach may be to either connect a removable (USB or similar) hard drive to the robot PC, or to unplug
+      the robot hard drive and nsert it into a PC or workstation.
+    - You can then use a tool such as CloneZilla or dd to write a backup image of your robot's hard drive onto another
+      hard drive.
+    - Alternatively, you can simply replace the robot computer's hard-drive, reserving the drive and installing a new one
```

Comment:
change to 'hard drive' to match how it is used on the rest of the page.

Reply:
Fixed